contrib.json

```diff
@@ -372,5 +379,53 @@
         [ "git branch -D {{pr.user.login}}-{{pr.head.ref}}",                   "Delete the local branch used for merging" ]
       ]
     }
+  },
+
+  "stage": {
+    "patch": {
```

Comment:
This updates the local stable and patch repos, merges stable into patch and pushes upstream to trigger the build.